README.rst

```diff
@@ -0,0 +1,80 @@
+==============================
+REANA Authentication VOMSproxy
+==============================
+
+About
+=====
+
+``reana-auth-vomsproxy`` provides a container image for creating and renewing a VOMS proxy certificate. The container image
+includes no additional logic or libraries, just the bare minimum to support the VOMS client. 
+The image is configured to support authentication for the four experiments at CERN; ALICE, ATLAS, CMS and LHCb.
+
+``reana-auth-vomsproxy`` was developed for use in the `REANA <http://www.reana.io/>`_ reusable research data analysis platform.
+
+Usage
+=====
+
+You can use ``reana-auth-vomsproxy`` as a base image, however it was built
+to be used as a sidecar container with the single purpose of refreshing the proxy certificate. 
+Once obtained, the certificate is shared with the main container using common namespaces. 
+
+The end users can ask for VOMS authentication by means of declaring voms: true, more information `here <http://docs.reana.io/advanced-usage/access-control/voms-proxy/>`_.
+
+If you would like to try it locally, you can run:
+
+.. code-block:: console
+
+   $ docker run -i -t --rm -v $HOME/foo:/root/.globus/ reanahub/reana-auth-vomsproxy:1.0.0 /bin/bash
+
+Your local directory ``/foo`` should contain your ``usertcert.pem`` and ``userkey.pem`` files. By default the VOMS client
+checks the directory ``/$HOME/.globus`` for the files needed. In this image that path results in ``/root/.globus/``.
+
+Inside the container a proxy can be obtained by specifying the Virtual Organization, for example via:
+
+.. code-block:: console
+
+  $ voms-proxy-init --voms cms
+
+You can build the ``reana-auth-vomsproxy`` image by passing the build arguments ``DATE`` in
+the format "YYYY-MM-DD" and ``VERSION`` as either a semantic versioning and/or the git SHA.
+
+Configuration
+=============
+
+Running the container and successfully obtaining a VOMS proxy certificate requires additional information and inputs:
+
+- Grid user certificate: ``${HOME}/.globus/usercert.pem``
+- Grid user key: ``${HOME}/.globus/userkey.pem``
+- Grid user password
+- Virtual organisation membership (e.g. for CMS: ``cms``)
+
+CHANGES
+=======
+
+Version 1.0.0 (2020-08-05)
+- Initial release
+
+Development
+===========
+
+If you would like to contribute to ``reana-auth-vomsproxy`` development, you can
```

Comment:
There is no `Makefile` right now... so perhaps delete this section.

(Or move parts from above saying "You can build image ... YYYY-MM-DD ..." here, because this is actually development  of new images, so to speak; so it rather belongs here in the Development section than in the Usage section.)